src/install/haproxy/linux/install-linux.mdx

```diff
@@ -0,0 +1,46 @@
+---
+componentType: default
+headingText: Download the  integration
+---
+
+1. Change directory to the integrations configuration folder by running:
+
+```shell
+cd /etc/newrelic-infra/integrations.d
+```
+
+2. Copy the sample configuration file by running:
+
+```shell
+sudo cp nri-apache-config.yml.sample nri-apache-config.yml
+```
+
+3. Edit the `nri-apache-config.yml` file. This example config file collects all metrics:
```

Comment:
I might add a little context here. I had to google how HAProxy relates to Apache. this might be given for the audience, but just in case I might add something like: 

2. Since HAProxy does X with Apache, you need to use the New Relic Apache blahblahblah to do the yadda yadda. 

(defer to you though -- could be a given)

Reply:
Maybe this is dependent on the install path being used, but I notice that yum refers to nri-haproxy and not nri-apache. since nri-haproxy does exist, why don't the other install paths use it?